addon_imps/storage/box_dot_com.py

```diff
@@ -13,7 +13,7 @@ class BoxDotComStorageImp(storage.StorageAddonImp):
     """
 
     async def get_external_account_id(self, auth_result_extras: dict[str, str]) -> str:
-        async with self.network.GET("/users/me") as _response:
+        async with self.network.GET("users/me") as _response:
```

Comment:
Can you document the "why" of this change. Putting it in the CR description is fine.

Reply:
The reason is that it raised exception. `ClientSession` expects path without / in the beginning